docs/src/main/sphinx/optimizer/pushdown.rst

```diff
@@ -127,13 +129,64 @@ performed, and instead Trino performs the aggregate processing.
                regionkey := tpch:regionkey
 
 Limitations
------------
+^^^^^^^^^^^
 
-Pushdown does not support a number of more complex statements:
+Aggregation pushdown does not support a number of more complex statements:
 
 * complex grouping operations such as ``ROLLUP``, ``CUBE``, or ``GROUPING SETS``
 * expressions inside the aggregation function call: ``sum(a * b)``
 * coercions: ``sum(integer_column)``
 * :ref:`aggregations with ordering <aggregate-function-ordering-during-aggregation>`
 * :ref:`aggregations with filter <aggregate-function-filtering-during-aggregation>`
 
+.. _join-pushdown:
+
+Join pushdown
+-------------
+
+Join pushdown allows the connector to delegate the table join operation to the
+underlying data source. This can result in performance gains, and allows Trino
+to perform the remaining query processing on a smaller amount of data.
+
+The specifics for the supported pushdown of table joins varies for each data
+source, and therefore for each connector.
+
+.. _limit-pushdown:
+
+Limit pushdown
+^^^^^^^^^^^^^^
+
+A :ref:`LIMIT or FETCH FIRST clause <limit-clause>` reduces the number of
```

Comment:
Let's change "clauses" to "clause" in the SELECT documentation and then we don't need to duplicate the text in all the links. I think it makes more sense as singular, since a query only has one (it's two alternatives for the same thing).

Reply:
Done